app/styles/_globals.scss

```diff
@@ -142,23 +142,16 @@ button {
   left: 0;
 }
 
-// We're using a polyfill for the upcoming CSS4 `:focus-ring` pseudo-selector.
-// This allows us to not have to override default accessibility driven focus
-// styles for buttons in the case when a user clicks on a button. This also
-// gives better visiblity to individuals who navigate with the keyboard.
-//
-// See:
-//   https://github.com/WICG/focus-ring
-//   Focus Ring! -- A11ycasts #16: https://youtu.be/ilj2P5-5CjI
-:focus:not(.focus-ring) {
+:not(:focus-visible) {
   outline: none;
 }
 
 // http://stackoverflow.com/questions/7538771/what-is-webkit-focus-ring-color
-:focus {
+:focus-visible {
   outline: auto;
   outline-color: var(--focus-color);
 }
+
```

Comment:
Does this make sense? Otherwise I was seeing the outline on _certain_ elements as I clicked them.

Reply:
whats focus-visable

Reply:
It makes sense but not intuitively. I think to make this clearer by skipping the negation above and just specifying that `:focus` does not apply an outline but `:focus-visible` does. And since `:focus-visible` implies `:focus` I think it makes more sense to declare the outline color on the `:focus` selector in case we have components which want to show an outline on pointer focus.

```scss
// By default we don't want to show the focus ring on elements unless the focus
// occurred as a result of keyboard navigation.
:focus {
  outline: none;
  outline-color: var(--focus-color);
}

:focus-visible {
  outline: auto;
}
```

Reply:
I like that! I had to move the `outline-color` line to the `:focus-visible` part in order to make it work as expected, but that's it.

Thank you! 💖